samples/ControlCatalog/Models/MathConverter.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Avalonia.Data.Converters;
+
+namespace ControlCatalog.Models
+{
+    public class MathConverter : IValueConverter
+    {
+        private static readonly List<char> _allOperators = new List<char> { '+', '-', '*', '/', '%', '(', ')' };
+
+        private static readonly List<string> _grouping = new List<string> { "(", ")" };
+        private static readonly List<string> _operators = new List<string> { "+", "-", "*", "/", "%" };
```

Comment:
It's better to use arrays instead of lists here. It eliminates excess `.ToArray()` calls.

Reply:
Thank you for your attention!

I decided to submit negative values ​​and this code stopped working. Therefore, I decided to make a simple option